pages/_app.js

```diff
@@ -1,10 +1,29 @@
 import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
+
+const fetcher = async (url) => {
+  const response = await fetch(url);
+  if (!response.ok) {
+    const error = new Error("An error occurred while fetching the data.");
+    // Attach extra info to the error object.
+    error.info = await res.json();
+    error.status = res.status;
+    console.log(error);
+    throw error;
+  }
+  return response.json();
+};
 
 export default function App({ Component, pageProps }) {
   return (
-    <>
-      <GlobalStyle />
+    <SWRConfig>
+      <GlobalStyle
+        value={{
+          fetcher,
+          refreshInterval: 5000,
```

Comment:
Do you really need a refreshInterval for the art pieces fetch? I am not sure, I think there is nothing changing on refresh, or?

Reply:
good point - removed it! ✅